From 702555c4ec428a5ad6e6be2a90850453be8f420b Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Wed, 20 Jan 2021 00:12:15 +0300
Subject: [PATCH 02/18] plat: rcar_sip_svc: Fix build error when
 PMIC_ROHM_BD9571 is disabled

This fixes unused "ret" variable error when building with PMIC_ROHM_BD9571=0.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 plat/renesas/rcar/rcar_sip_svc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/renesas/rcar/rcar_sip_svc.c b/plat/renesas/rcar/rcar_sip_svc.c
index 44307ca..d503bc7 100644
--- a/plat/renesas/rcar/rcar_sip_svc.c
+++ b/plat/renesas/rcar/rcar_sip_svc.c
@@ -23,7 +23,9 @@ static uintptr_t rcar_sip_handler(unsigned int smc_fid,
 			void *handle,
 			u_register_t flags)
 {
+#if PMIC_ROHM_BD9571
 	int32_t		ret;
+#endif
 	int32_t		board_ret;
 	uint32_t	board_type = 0U;
 	uint32_t	board_rev = 0U;
-- 
2.7.4

